README.md

```diff
@@ -1,4 +1,4 @@
-# Iotics API
+as their corresponding service# Iotics API
 
 Create Data Mesh. Use interoperable digital twins to create data interactions and build powerful real-time data products. This repository contains protocol buffers specification for Iotics API.
```

Comment:
can you pls remove "Create Data Mesh". 

also

"this repository contains protocol buffers and gRPC service definitions for the Iotics API"